src/BlazorWebView/src/Maui/iOS/IOSWebViewManager.cs

```diff
@@ -164,9 +164,10 @@ private static UIAlertAction AddCancelAction(UIAlertController controller, Actio
 
 				if (cancelAction != null)
 					AddCancelAction(controller, () => cancelAction(controller));
-
+#pragma warning disable CA1416 // TODO:  'UIApplication.Windows' is unsupported on: 'ios' 15.0 and later
 				GetTopViewController(UIApplication.SharedApplication.Windows.FirstOrDefault(m => m.IsKeyWindow)?.RootViewController)?
```

Comment:
@TanayParikh - I think this was code maybe you added for JavaScript alerts? Any idea what we should do here?

Reply:
Looks like this was added in January to "implement WKUIDelegate into WKWebView in BlazorWebView".

https://github.com/dotnet/maui/pull/4158/files

Is this warning essentially saying this functionality is broken in iOS 15? If so that seems pretty bad (RC3 candidate even).

Reply:
Created https://github.com/dotnet/maui/issues/6020 to investigate and tentatively put into the RC3 milestone.

Reply:
> Is this warning essentially saying this functionality is broken in iOS 15? 

The warning is `'UIApplication.Windows' is unsupported on: 'ios' 15.0 and later, 'maccatalyst' 15.0 and later.` By design the unsupported attribute should be added to the APIs that is removed/not work anymore. But from the previous suggestions the MAUI team given: some of the APISs annotated with unsupported attribute were deprecated APIs but still works. So yes it is better to investigate if it is broken in or not and take action. With that reason I mostly suppressed the warnings with `// TODO: ... ` comment